components/Buttons/FavSeenContainer/ButtonFavorite/index.js

```diff
@@ -1,10 +1,23 @@
 import { ButtonFav, ButtonFavStyle } from "./ButtonsStyle";
+import { toast } from "react-toastify";
+import { ProgressBar } from "react-toastify";
+import { StyledToastContainer } from "..";
 import Image from "next/image";
 import React, { useState, useEffect } from "react";
 import Star from "./star.svg";
 import StarFilled from "./star-filled.svg";
+import "react-toastify/dist/ReactToastify.css";
```

Comment:
please move your styles somewhere else, we don't use CSS files except from styles.js (global styles)